.travis.yml

```diff
@@ -30,10 +30,11 @@ jobs:
     - git config --global user.email "releases@ladybug.tools"
     - git config --global user.name "ladybugbot"
     - npx semantic-release
-    provider: pages
-    skip_cleanup: true
-    github-token: $GH_TOKEN
-    keep-history: false
-    on:
-      branch: master
-    local_dir: docs/_build/
\ No newline at end of file
+    deploy:
```

Comment:
do we need deploy twice? isn't the stage already set to deploy in line 20?

Reply:
Nah this is a specific marker to denote the deployment scripts in travis at any stage.